physics/module_bl_mynn.F90

```diff
@@ -5362,6 +5362,27 @@ SUBROUTINE mynn_mix_chem(kts,kte,i,     &
     rhoz(kts)  =rho(kts)
     rhoinv(kts)=1./rho(kts)
     khdz(kts)  =rhoz(kts)*dfh(kts)
+
+    khdz_old  = khdz(kts)
+    khdz_back = pblh * 0.15 / dz(kts)
+    !Enhance diffusion over fires
+    IF ( fire_turb ) THEN
+       IF ( pblh < pblh_threshold ) THEN
+          IF ( emis_ant_no > no_threshold ) THEN
+!             khdz(kts) = MAX(khdz(kts),khdz_back)
+              khdz(kts) = MAX(1.1*khdz(kts), sqrt((emis_ant_no / no_threshold))/dz(kts)*rhoz(kts))   ! JLS 12/21/21
+          ENDIF
+          IF ( frp > frp_threshold ) THEN
+              !kmaxfire = ceiling(log(curr_frp))  ! JLS 12/21/21  - need to bring in curr_frp
+              kmaxfire = ceiling(log(frp))
+              IF (k .le. kmaxfire) THEN    ! JLS
+!             khdz(kts) = MAX(khdz(kts),khdz_back)
+              khdz(kts) = MAX(1.1*khdz(kts),((log(frp))**2.- 2.*log(frp)) / dz(kts)*rhoz(kts)) ! JLS 12/21/21
+              ENDIF ! JLS
+          ENDIF
+       ENDIF
+    ENDIF
+
     DO k=kts+1,kte
```

Comment:
The eddy diffusivity is always == 0 at kts (=1), which is why this was originally removed, so it is only enhance above the surface. I had this discussion with Jordan a while back. I think this section should be removed.

Reply:
Joe, this part was deleted in the updated PR.

Reply:
excellent